user-service/src/main/java/com/ecommercepractice/userservice/models/User.java

```diff
@@ -62,10 +62,11 @@
     @ApiModelProperty(value = "User's email")
     private  String email;
 
-    public User(String password, String firstName, String secondName, boolean isActive, String email){
+
```

Comment:
Use Lombok to avoid

Reply:
It's necessary. It's necessary a constructor with all but userId, I did a search and I found that the only way to accomplish this with Lombok is assign notNull and final to the fields in order to use the @RequiredArgsConstructor.